### PR TITLE
Add display configuration to start Plover minimized

### LINKED ISSUE #475: Start-Minimized Option
Classification: Feature (New)

Reproducibility: N/A

## Summary

When adding a translation, the Plover window comes to front unless minimized. I often forget to minimize manually on startup, so having an option to always start minimized would be extremely useful.

## Steps to Reproduce

I check the "start minimized" box in config.

## Expected Results

Plover starts minimized.

## Actual Results

Plover has to be manually minimized to keep the main window to pop up when adding translations.

If helpful, submit screenshots of the issue to help debug. Your `plover.log` or `strokes.log` may also be useful.

## Version
2.5.8

## Notes
Workaround: Minimize Plover before adding any translations.

## Configuration
???

### PR DESCRIPTION
**Pull request by morinted**
Fix #475, add config to start Plover minimized.

Screenshot of the configuration option to start Plover minimized

### CODE REVIEW

**Comment by morinted** [comment]
I've only tried this on my KDE install, I'll do OSX tomorrow. @stenoknight You can try with the AppVeyor artifact: https://ci.appveyor.com/api/buildjobs/pdrikwlin253d2ms/artifacts/dist/plover-3.0.0+12.ga45aad5.exe

**Comment by stenoknight** [comment]
X'D

File: plover/gui/main.py [hunk 1]
```diff
@@ -51,9 +51,9 @@ def __init__(self, config):
     def OnInit(self):
         """Called just before the application starts."""
         # Enable GUI logging.
-        from plover.gui import log as gui_log
```
Comment (benoit-pierre) [inline plover/gui/main.py]: Mistake?

**Comment by benoit-pierre** [comment]
Works fine on Ubuntu and Arch Linux with Awesome. Idem on Windows 10.

File: plover/gui/main.py (hunk 1 above)
Comment (morinted) [inline plover/gui/main.py]: Ah, it didn't look used, but does this import actually do something?

Comment (benoit-pierre) [inline plover/gui/main.py]: It initializes the GUI log: adding the platform specific log handler.

**Comment by morinted** [comment]
On OS X, frame.Show() starts raises the window…

I tried removing set top app, nada. If I place an iconize event after the show, the Plover window raises then minimizes. If I set iconized false, then show, then set iconized false again…Plover starts in the dock, raises to display itself in its glory, and then minimizes again (which is very annoying on Mac with the genie lamp animation).

If I don't call show, then Plover is in the dock, but you can see that its mainframe isn't displayed…

**Comment by morinted** [comment]
@benoit-pierre The best I can get on OSX is to show the window then immediately iconize it. The same code looks silly on Winux because you see a flash of the window and lose focus for a sliver of time.

The solution is as committed... maybe you could suggest something a little more elegant?

```python
        osx = sys.platform.startswith('darwin')
        if not osx:
            frame.Iconize(self.config.get_start_minimized())
        frame.Show()
        if osx:
            frame.Iconize(self.config.get_start_minimized())
```

**Pull request merged**